src/java.desktop/macosx/classes/sun/lwawt/macosx/CAccessible.java

```diff
@@ -184,6 +184,11 @@ public void propertyChange(PropertyChangeEvent e) {
                     if (thisRole == AccessibleRole.CHECK_BOX) {
                         valueChanged(ptr);
                     }
+
+                    // Do send radio button state changes to native side
+                    if (thisRole == AccessibleRole.RADIO_BUTTON) {
+                        valueChanged(ptr);
+                    }
```

Comment:
Have you checked to see if this would negatively impact voice over and other accessibility tools?

Reply:
I checked with voice over and it does announce the correct state of the radiobutton.
In fact after this fix, VO announces it's selected state as well which was not announced before without this fix.

What are the other a11y tools we can check?

Reply:
TNX.